README.rst

```diff
@@ -0,0 +1,57 @@
+=================
+encrypted-secrets
```

Comment:
```suggestion
Encrypted secrets
```

Just cause it's a title

Reply:
It's also the module name though so... :shrug: